### ISSUE TITLE
Multiple databases support

### ISSUE THREAD

**Issue by jibin-tj** [blue-color/nodejdbc#15]
Can I connect to an Oracle DB to fetch some value and insert it to some Teradata DB ?

**Comment by ceyhunkerti** [comment]
yes,
you'll need two separate connections

**Comment by jibin-tj** [comment]
How can we connect to multiple vendors(oracle and teradata) using multiple
connections? Did you mean multiple nodejdbc objects and getting connections
from each?
I tried this but when i try to initialize the second nodejdbc object with
new driver,url etc ,its throwing classdef not found.

**Comment by ceyhunkerti** [comment]
yes with `new NodeJDBC(config)`
do you have jdbc driver for both oracle and teradata ?

**Comment by jibin-tj** [comment]
Yes,I have both drivers. But the second new NodeJDBC(config) is unable to
find the driver class and throws class def not found error
I am wondering whether this is because already a driver class is loaded .

**Comment by ceyhunkerti** [comment]
can you share the part of code here ?

**Comment by jibin-tj** [comment]
Hi
Here is the part of the code.

`router.get('/', function(req, res, next) {
	logger.info("Request hit");
	var teradataDb = new NodeJDBC(teradataDbOptions);
	var oracleDb = new NodeJDBC(oracleDbOptions);
	res.send("OK");
}`

Individually it works fine but not both .I have tried switching the order of initilization. Only the first Db is initialized and a classDefNotFound error is thrown for second item.

**Comment by jibin-tj** [comment]
Hi
I figured out that it works fine when all the jars(ojdbc6 also) are added in libs in teradataOptions itself (if we are initializing it first)

**Comment by ceyhunkerti** [comment]
That's wierd it should't be necessary to include all. Your above code is correct.
We can leave the issue OPEN. I'm planning to migrate sources to es6, i'll have a closer look to this issue during that time.

thanks..
